server/models/notification.js

```diff
@@ -0,0 +1,31 @@
+const mongoose = require("mongoose");
+
+const notificationSchema = new mongoose.Schema({
+  userId: {
+    type: mongoose.Schema.Types.ObjectId,
+    required: true,
+    ref: "User",
+  },
+  title: {
+    type: String,
+    required: true,
+  },
+  description: {
+    type: String,
+    required: true,
+  },
+  readunread: {
+    type: String,
```

Comment:
```suggestion
    type: String,
    default: false,
```
You can add a default of 'false' as all notifications will be unread at first :)